client/flutter/lib/api/question_data.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:WHOFlutter/api/content_loading.dart';
+import 'package:flutter/cupertino.dart';
+
+class QuestionData {
+  static Future<List<QuestionItem>> yourQuestionsAnswered(
+      BuildContext context) async {
+    var bundle =
```

Comment:
does this need to be wrapped in a `try`/`catch` to handle the exception thrown if there's no content bundle? or is that exception handled elsewhere?

Reply:
yes agree @patniemeyer

Reply:
👍